### ISSUE TITLE
read meta-url from variable

### ISSUE THREAD

**Issue by sedlund** [juicedata/juicefs#5226]
**What would you like to be added**:

`juicefs` cli should read meta-url from an environment variable by if set.
I suggest JFS_URL

If the meta-url is specified on the command line, it should take precedence.

**Why is this needed**:

it makes the cli nicer to use not having to specify the url each time it is required.

**Comment by solracsf** [comment]
Agree, but it would make more sense a `META_URL`, as `META_PASSWORD` already exists.

**Comment by sedlund** [comment]
agree, would like to see all juicefs variables prefixed as is common elsewhere -- JFS_ or JUICEFS_

**Comment by suzaku** [comment]
So if I run `juicefs mount xxx`, `xxx` would be interpreted as the mount point while the URL is read from the environment variable?
What if I only specify the URL and miss the mount point?

**Comment by sedlund** [comment]
> So if I run juicefs mount xxx, xxx would be interpreted as the mount point while the URL is read from the environment variable?

Yes.   If $JUICEFS_META_URL is set, it fits a validation schema and `xxx` is a directory.

> What if I only specify the URL and miss the mount point?

Following the same steps above, the URL fits validation URL schema then it will error on no mount point specified.

**Comment by davies** [comment]
It's better to keep the command arguments consistent.

The `sync` command read the meta-url from environment variable, which use the volume name as the name, for example,

```
myfs=redis://localhost/  juicefs sync jfs://myfs/prefix s3://mybucket/prefix
```

It support multiple volumes in the same time, maybe we can do the same thing here:

```
myjfs=redis://localhost/  juicefs mount myjfs /mnt
```

But, we use `redis://` as the default schema, and the `myjfs` will be treated as hostname rather than a variable name.

**Comment by davies** [comment]
There is not much we can do without breaking existing command options, close it for now.
